.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,108 @@
+---
+name: Bug report
+description: File a bug report
+title: "bug: "
+labels: bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug!**
+
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+      Also test if the latest release is affected too.
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with teraform module from the main branch on GitHub, Y
+      breaks in a way Z under the env E. Here are all the details I know
+      about this problem...
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Issue Type
+    description: >
+      Please select the single available option in the drop-down.
+
+      <details>
+        <summary>
+          <em>Why?</em>
+        </summary>
+
+        We would do it by ourselves but unfortunatelly, the curent
+        edition of GitHub Issue Forms Alpha does not support this yet 🤷
+
+
+        _We will make it easier in the future, once GitHub
+        supports dropdown defaults. Promise!_
+
+      </details>
+    options:
+    - Bug Report
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Terraform Version
```

Comment:
label, description and placeholder in this template should be adjusted to a bug for helm chart